README.md

```diff
@@ -1,43 +1,24 @@
-# How to set up Grafana on Clever Cloud
+# How to set up Grafana on Clever-Cloud
 
-1. create a new NodeJs application linked to a fork of this repo
-2. in your application environment variables add ```GF_SERVER_HTTP_PORT=8080```, ```GOPATH=/home/bas/<app_id>/go_home``` where <app_id> is the id of your application on clever cloud (found in the top right corner of the console) and ```GRAFANA_VERSION=v7.3.5``` (or any other tag in the official Grafana repo)
-3. you can now build and start the application, be aware however that if you restart it your data will not be persisted (see next section to learn how to persist data)
+1. Use a fork of this repository as a Node.js application. It will download and run Grafana using the root `package.json` file. The install script `build.sh` will download grafana and its plugins and the start one `run.sh` will start the grafana-server binary.
+2. In your application environment variables add ```GF_SERVER_HTTP_PORT=8080```.
+3. Add ```GRAFANA_VERSION``` and ```GRAFANA_SHA_256``` environment variables. The grafana dowload page is [here](https://grafana.com/grafana/download).
+4. Check Node.js version used in `package.json`, for instance Grafana 7.5.5 expects Node.js 14 (see Grafana's [package.json](https://github.com/grafana/grafana/blob/v7.5.5/package.json#L317)).
+5. To install custom Grafana plugin, you can set them directly in `GRAFANA_PLUGINS` environnement variable split by a `,`, for example with the value: `plugin,other_one:0.0.2,https://github.com/ovh/ovh-warp10-datasource.git`. It will install the latest version of `plugin`, the version `0.0.2` of `second_one` using the grafana-cli commands. And for the `ovh-warp10-datasource` plugin, `git clone` is used instead, as it's done for any plugin starting by `http`.
+6. Add `GF_PLUGIN_DIR` environnement variable with `/home/bas/APP_ID/grafana/data/plugins` to use local cloned plugins.
```

Comment:
Didn't you manage to make this work with a relative path? If we could avoid having the app id written there, it could be great.